idx/cassandra/meta_records.go

```diff
@@ -122,16 +130,21 @@ func (c *CasIdx) loadMetaRecords() {
 			toLoad[orgId] = batchId
 		}
 	}
+
 	var err error
 	if err = iter.Close(); err != nil {
 		log.Errorf("Error when loading batches of meta records: %s", err.Error())
+		c.sessionLock.RUnlock()
 		return
 	}
+	c.sessionLock.RUnlock()
```

Comment:
it should be fine to release the `sessionLock` before `iter.Close()`, because that's not accessing the session anymore